### PR TITLE
Update Traffic/Metrics/Measurements docs

### PR DESCRIPTION
**Pull request by EdwardAngert**
## Summary

**[Traffic Limits and Overages
](https://pantheon.io/docs/traffic-limits)** and **[Metrics in the Site Dashboard](https://pantheon.io/docs/metrics)** - Overhauled the docs related to traffic and metrics to outline Pantheon's policies more transparently and more directly. [PR 5817](https://github.com/pantheon-systems/documentation/pull/5817)

## Effect
The following changes are already committed:

- Includes and updates changes proposed in #5715 
- adds back info from #5239 (which was the result of customer and CSE feedback)
- update definitions partial

## Remaining Work
The following changes still need to be completed:

- [x] merge google docs in
- [ ] hold for release

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Update Status Report
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | [Live](https://pantheon.io/docs) | [pr-5817](https://pr-5817--pantheon-docs.my.pantheonfrontend.website/docs) | Difference |
| --- | --- | --- | --- |
| Performance Score | `51` | `53` | `2` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `98` | `98` | `0` |

Lighthouse Audit Reports:

- [`pr-5817` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/22157/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/22157/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)


[Preview Environment](https://pr-5817--pantheon-docs.my.pantheonfrontend.website/docs)

File: source/content/traffic-limits.md [hunk 1]
```diff
@@ -1,75 +1,76 @@
...
+**Exclusions**. The traffic measurement model above describes those interactions specifically excluded. Inherent in our model is customer feedback. [Contact support](/support) to request that Pantheon consider additional exclusions if you experience issues with your site and any of the following:
 
-### 404s and Other Client Errors
-Pantheon only counts pages returned, considered [`200` level](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#2xx_Success), in traffic metrics. The Platform does not count "redirects" which are returned as [`300` level](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#3xx_Redirection) and "client errors," which are returned as [`400` level](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#4xx_Client_errors), as part of plan traffic limits.
+- **Static Assets**: Static requests (images, PDFs, CSS, JS, etc) are not included in traffic metrics reporting on the Site Dashboard. Under regular CMS use cases, these requests supporting requests to render HTML pages for users with browsers do not count as additional visits.
```
Comment (alexfornuto) [inline source/content/traffic-limits.md:35]: I've attempted to clarify this bullet point, but it's still lacking in clarity. I'd like to make more apparent what's being discussed here, between static assets being called directly, vs loaded as part of an html page rendering.

File: source/content/traffic-limits.md [hunk 2]
```diff
@@ -1,75 +1,76 @@
...
-Analytics suites (like Google Analytics) are measuring fundamentally different things when compared to Pantheon's request log. While analytics suites focus on measuring _visits_, our request log more comprehensively measures _traffic_.
 
-We track every single request to the platform, whereas analytics tools will typically only track complete "pageviews" where an HTML page including a tracking snippet is completely loaded by a browser and can fire off a subsequent request to the analytics platform.
+Website traffic is an important indicator of a successful website. Analytics suites (e.g. Google Analytics, Similarweb, Mixpanel) each serve a different purpose from Pantheon’s Site Dashboard.
+
+Pantheon tracks every single request to the platform. In contrast, analytics tools will typically only track complete "pageviews" on an HTML page containing a tracking snippet that can fire off a subsequent request to the analytics platform.
+
+This table shows some of the reasons why traffic in the Dashboard may differ from your analytics suite:
+
+<Partial file="traffic-analytics-table.md" />
```
Comment (alexfornuto) [inline source/content/traffic-limits.md:73]: The table in this partial and the bulleted list below are redundant. Consider choosing one and removing the other.

Comment (EdwardAngert) [reply source/content/traffic-limits.md:73]: letting this stand for now as the table serves as more of a quick reference visual

**Pull request merged**